modules/comment-likes/admin-style.css

```diff
@@ -0,0 +1,55 @@
+.vers img {
```

Comment:
This feels somewhat overly broad styling -- probably fine, but may conflict with other plugins down  the road.